pkg/query/querier.go

```diff
@@ -140,9 +150,12 @@ func newQuerier(
 }
 
 // matchStore returns true iff the given store may hold data for the given label matchers.
```

Comment:
not your mistake, but `s/iff/if` (:

Reply:
He, this comes up every once in a while: https://en.wikipedia.org/wiki/If_and_only_if

I use the abbreviation sometimes if I think it should be clear but don't want the verbosity.